client_test.go

```diff
@@ -193,8 +194,9 @@ func TestPutSecurity(t *testing.T) {
 }
 
 type testDocument struct {
+	ID    string `json:"_id,omitempty"`
 	Rev   string `json:"_rev,omitempty"`
-	Field int64  `json:"field"`
```

Comment:
Why are you changing this?

Reply:
I didn't see the point of having this field as an int64 and have all clients do things like `check(t, "doc.Field", int64(999), doc.Field)` when you could just use int, it was just a question of noise.